@xen-orchestra/lite/src/App/index.tsx

```diff
@@ -1,25 +1,39 @@
 import Cookies from 'js-cookie'
 import React from 'react'
 import styled from 'styled-components'
-import { IntlProvider } from 'react-intl'
-import { HashRouter as Router, Switch, Route, Link } from 'react-router-dom'
-import { Map } from 'immutable'
+import { FormattedMessage, IntlProvider } from 'react-intl'
+import { HashRouter as Router, Switch, Route } from 'react-router-dom'
 import { withState } from 'reaclette'
 
 import Button from '../components/Button'
 import IntlMessage from '../components/IntlMessage'
+import ListObjects from './ListObjects'
 import messagesEn from '../lang/en.json'
-import PoolTab from './PoolTab'
 import Signin from './Signin/index'
 import StyleGuide from './StyleGuide/index'
 import TabConsole from './TabConsole'
-import XapiConnection, { ObjectsByType, Vm } from '../libs/xapi'
+import XapiConnection, { ObjectsByType } from '../libs/xapi'
+
+const LeftView = styled.div`
+  background: #f5f5f5;
+  float: left;
```

Comment:
Avoid `float`. You can use flexbox or css-grid to position your panels on the page.